bundles/org.openhab.binding.openweathermap/src/main/java/org/openhab/binding/openweathermap/internal/dto/onecallhist/Weather_.java

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.openweathermap.internal.dto.onecallhist;
```

Comment:
What is the difference between this class and the other `Weather` class? (see https://github.com/openhab/openhab-addons/pull/9062/files#diff-9a8c2612d60ade9bb74e96978e5aba5d07bd90f08579e2001039c9c212e79cd7)

Reply:
Also removed